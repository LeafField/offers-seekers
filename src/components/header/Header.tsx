import { FC } from "react";
import styles from "./Header.module.css";

const Header: FC = () => {
  return (
    <header className={styles.header}>
      <h1 className={styles.siteTitle}>求人・求職者数</h1>
    </header>
  );
};

export default Header;
