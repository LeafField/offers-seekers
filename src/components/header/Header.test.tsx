import { expect, test, describe } from "vitest";
import { screen, render } from "@testing-library/react";
import Header from "./Header";

describe("Headerの単体テスト", () => {
  test("コンポーネントが正しくレンダリングされている", () => {
    render(<Header />);
    expect(
      screen.getByRole("heading", { name: "求人・求職者数" }),
    ).toBeDefined();
  });
});
