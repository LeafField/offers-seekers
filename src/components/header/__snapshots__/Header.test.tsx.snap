// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Headerの単体テスト > スナップショットテスト 1`] = `
<div>
  <header
    class="_header_69e643"
  >
    <h1
      class="_siteTitle_69e643"
    >
      求人・求職者数
    </h1>
  </header>
</div>
`;
